src/dashboard/Data/Browser/Browser.react.js

```diff
@@ -356,7 +380,7 @@ class Browser extends DashboardView {
 
     query.limit(MAX_ROWS_FETCHED);
 
-    let promise = query.find({ useMasterKey: true });
+    let promise = query.find({ useMasterKey: useMasterKey });
```

Comment:
@mtrezza here is usage of `useMasterKey` state.

Reply:
So assuming another user only affects what you see in the dashboard, but not other rights such as deleting or modifying data?

Reply:
Yes, only fetch/view. Didn't need other functionalities at the moment, but I assume other rights could also be taken into consideration down the line.

Reply:
Got it, looks good to me.

Reply:
So what do you think, better to stay `browse as user` or change it to `assume user`? Somehow after this conversation `browse as user` sounds better to me.

Reply:
I don't see much difference because "browsing” can also be understood as reading and writing. I think however you name it, it stays counter-intuitive that you read as one user but write as another user. Mentioning this in the docs should be fine. Have you looked into how much more work it would be to apply the `useMasterKey` option also to the writing commands?